tests/integration/linux_bridge_test.py

```diff
@@ -139,6 +148,14 @@ def test_remove_bridge_and_keep_slave_up(bridge0_with_port0, port0_up):
     assert 1 == len(current_state[Interface.KEY])
 
 
+def test_create_vlan_as_slave_of_linux_bridge(setup_vlan_iface):
+    bridge_name = TEST_BRIDGE0
+    port_name = setup_vlan_iface[Interface.KEY][0][Interface.NAME]
+    bridge_state = _create_bridge_subtree_config((port_name,))
+    with _linux_bridge(bridge_name, bridge_state):
+        assert_mac_address(setup_vlan_iface)
```

Comment:
Should use `assertlib.assert_state` instead of checking mac address here.

Reply:
Why is assert_state a better option there (your comments)? assert_mac_address checks whether all the macs in the state are the same or not and that's what master-slave have. So I was trying to check that whether all the interfaces share the same MAC or not than comparing them against the current state(what you had recommended by using assert_state)

Reply:
This test is targeting to enslave vlan to linux bridge. Checking mac address is not it should cover.

We do single thing and do it best.